vm/systemSmartContracts/eei.go

```diff
@@ -141,6 +136,7 @@ func (host *vmContext) Transfer(
 	_ = senderAcc.BalanceDelta.Sub(senderAcc.BalanceDelta, value)
 	_ = destAcc.BalanceDelta.Add(destAcc.BalanceDelta, value)
 	destAcc.Data = append(destAcc.Data, input...)
+	destAcc.GasLimit += gasLimit
```

Comment:
The equivalent `Transfer()` function in Arwen has this line as well, but without the +, like this: `destAcc.GasLimit = gasLimit`. 

I don't know if it's good to perform multiple transfers to the same `destAcc` in the same `vmOutput`. If `destAcc` is a SC, then `+= gasLimit` makes sense, but `destAcc.Data = append(...)` might cause problems. 

No change requested here, but multiple transfers to the same `destAcc` needs extra discussion.

Reply:
yes. agreed. I think only one transfer should be possible. in case of Arwen we should make several output if there are several transfers.